docs/articles/documentation/using-testcafe/installing-testcafe.md

```diff
@@ -50,3 +57,53 @@ You can use the local version of TestCafe from a [command line](command-line-int
     ```bash
     testcafe chrome tests/
     ```
+
+## Using TestCafe Docker Image
+
+TestCafe provides a preconfigured Docker image with TestCafe, Chromium and Firefox installed,
+so that you can avoid the need to install browsers or the testing framework on the server.
+
+To learn how to install Docker to your system, see [Install Docker](https://docs.docker.com/engine/installation/).
+
+After Docker is installed, download the TestCafe Docker image from its repository.
+
+```sh
+docker pull testcafe/testcafe
+```
+
+The command above installs the stable version of the image. If you need an alpha version, run the following command.
+
+```sh
+docker pull testcafe/testcafe:alpha
+```
+
+Now you can run TestCafe from the docker image.
+
+```sh
+docker run -v ${TEST_FOLDER}:/tests -it testcafe/testcafe ${TESTCAFE_ARGS}
+```
+
+This command takes the following parameters:
+
+* `-v ${TEST_FOLDER}:/tests` - maps the `TEST_FOLDER` directory on the host machine to the `/tests` directory in the container. You can map any host directory to any container directory.
```

Comment:
`You can map any host directory to any container directory.` - colon here instead of period.